pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md

```diff
@@ -24,8 +25,8 @@ This release provides new features and enhancements to improve the user experien
 If you have DC/OS deployed in a production environment, see [Known Issues and Limitations](#known-issues-and-limitations) to see if any potential operational changes for specific scenarios apply to your environment.
 
 - Updated DC/OS UI to master+v2.150.2.
-- Updated to Mesos ??
-- Updated Marathon to 
+- Updated to Mesos ???
```

Comment:
This page seems out of data. Lets use data from https://github.com/dcos/dcos/blob/master/CHANGES.md as the starting point for release notes?